app/services/tts_service.rb

```diff
@@ -9,18 +9,34 @@ class TtsService < ApplicationService
   # TODO: make configurable
   #@@tts_url = 'http://127.0.0.1:8000/v0/speech'
 
-@@tts_url = 'https://api.grammatek.com/tts/v0/speech'
+  @@tts_url = 'https://api.grammatek.com/tts/v0/speech'
+
+@@voices_dict = {'Álfur' => 'Alfur',
+                 'Álfur (v2)' => 'Alfur_v2',
+                 'Diljá' => 'Dilja',
+                 'Diljá (v2)' => 'Dilja_v2',
+                 'Bjartur' => 'Bjartur',
+                 'Rósa' => 'Rosa',
+                 'Karl' => 'Karl',
+                 'Dóra' => 'Dora' }
 
 # max chars allowed in input text (for now)
 @@max_chars = 10000
 # a directory where the response audio is written to as .mp3
 @@tmp_audio_path = "./app/assets/audio/"
 
-  def initialize(text, format, filename)
+  def initialize(text, format, filename, duration, voice)
+    if @@voices_dict.key?(voice)
+      voice_id = @@voices_dict[voice]
+    else
+      # Let's not make the whole thing break, but use a default voice
+      voice_id= "Alfur"
+    end
+    p "=================== USING VOICE: #{voice_id} ======================"
```

Comment:
Shouldn't this be `Rails.logger.debug` ?

Reply:
probably ...